.github/workflows/test.yml

```diff
@@ -16,7 +16,13 @@ jobs:
       id: go
 
     - name: Check out code into the Go module directory
-      uses: actions/checkout@v1 
+      uses: actions/checkout@v1
+    - uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go- 
```

Comment:
`Install dependencies` takes only 10 seconds. I don't think we need to use cache now.
![image](https://user-images.githubusercontent.com/2253692/129018867-c6f7dad5-3522-4966-8d50-d8ace57b66d0.png)

Reply:
Done :) !